main/index.js

```diff
@@ -31,14 +37,11 @@ require('electron-debug')({
 
 autoUpdater.logger = log
 autoUpdater.logger.transports.file.level = 'info'
-log.info('App starting...')
+
 
 // To prevent garbage collection of the windows
 let windows = null
 
-// Set the application name
-app.name = 'OONI Probe'
```

Comment:
Why was this removed?

Reply:
We will use the value `productName` in `package.json` to keep it consistent. It already uses the same value.